src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const getDeclarationAsStrings = sourceString
+    .split(';')
+    .map((rawDeclaration) => rawDeclaration.trim());
+  const getArrayOfDeclarationEntries = getDeclarationAsStrings.map(
+    (preparedDeclaration) =>
+      preparedDeclaration
+        .split(':')
+        .map((preparedDeclarationToTrim) => preparedDeclarationToTrim.trim()),
+  );
+
+  return getArrayOfDeclarationEntries.reduce((acc, declarationEntry) => {
+    if (declarationEntry.length === 1) {
+      return acc;
```

Comment:
Checking the length of `declarationEntry` to be exactly 1 to skip the iteration is not entirely correct. If there's an empty declaration (resulting in an empty array), it won't be skipped. You should check if the first element of the `declarationEntry` array is an empty string instead.